runsc/boot/config.go

```diff
@@ -112,6 +113,20 @@ func MakeWatchdogAction(s string) (watchdog.Action, error) {
 	}
 }
 
+// MakeRefsLeakMode converts type from string
+func MakeRefsLeakMode(s string) (refs.LeakMode, error) {
+	switch strings.ToLower(s) {
+	case "disabled":
+		return refs.NoLeakChecking, nil
+	case "warning":
```

Comment:
Given the confusion, I think this might be a bad name. Before making this part of the public API, maybe we should consider renaming it?

I suggest:
disabled
log-names
log-traces

Adding log to the names allows other options in the future such as panic.

Reply:
@iangudger Addressed. Thank you